client/checkout/blocks/payment-methods-logos/style.scss

```
@@ -53,3 +53,9 @@
 		box-shadow: 0 0 0 1px rgba( 0, 0, 0, 0.1 );
 	}
 }
+
+.payment-methods--logos-popover {
```

An alternative to using `!important`

```suggestion
  .payment-method-label .payment-methods--logos ~ .logo-popover.payment-methods--logos-popover > img {
        object-position: center;
  }
```
This approach uses only WooPayments-scoped classes and the [sibling combinator](https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Styling_basics/Combinators#subsequent-sibling_combinator).

This works, but I wonder if this is better solved upstream in WooCommerce Blocks? The issue is that WooCommerce's rule:
  ```css
  .wc-block-checkout__payment-method ... .wc-block-components-radio-control__label img
  ```
  is too broad - it affects all images inside payment labels, including popover content. So I'd suggest to open an issue to make their selector more specific.  Their rule should probably only target direct payment method icons, not nested components like popovers.

 In the meantime, since we need this fix now, we can keep `!important` or use my suggestion.

---

Great, thank you. I'm going to open an issue upstream. I'll merge it as is for now, just to make it right for our users.

---

created this issue https://github.com/woocommerce/woocommerce/issues/61725